widgets/src/lib/types.ts

```diff
@@ -20,7 +20,9 @@ type Failure = {
 export type Step = {
 	type: RowColumnType;
 	value: string | null;
+	rerun_url: string;
 	created_at: Date | null;
+	occurred_at: Date | null;
```

Comment:
```suggestion
	failed_at: Date | null;
```

may be a better name.